src/components/AddInlineTrigger.tsx

```diff
@@ -0,0 +1,22 @@
+import { ComponentProps, forwardRef } from 'react';
+import styled from 'styled-components';
+import { IconPlusCircleOutline } from '@taskany/icons';
+
+import { InlineTrigger } from './InlineTrigger';
+
+const StyledInlineTrigger = styled(InlineTrigger)`
+    margin-left: 5px; // 24 / 2 - 7 center of UserPic and center of PlusIcon
+    height: 28px; // Input height
+`;
+
+interface AddInlineTriggerProps {
+    text: string;
+    onClick: ComponentProps<typeof InlineTrigger>['onClick'];
+    icon?: React.ReactNode;
+}
+
+export const AddInlineTrigger = forwardRef<HTMLDivElement, AddInlineTriggerProps>(
```

Comment:
But why? Why you don't using default `InlineTrigger` component?

Reply:
became a part of UserEditableList